src/profile.js

```diff
@@ -89,15 +168,35 @@ import {withRouter} from 'react-router-dom'
 
             <input type="text" name="name" value = {this.state.name} onChange={this.handleChange} maxlength="30" />
 
-            <p>Age:</p>
+            <p>Birthday:</p>
 
             <input
-              type="number"
-              name="age"
-              min="18"
-              max="100"
+              type="int"
+              name="birthday_yr"
+              min="1920"
+              onChange={this.handleChange}
+              value = {this.state.birthday_yr}
+              placeholder="YYYY*"
+            />
+
```

Comment:
maybe instead of inputting the year/date/month separately you can just have an input of type date